src/typeSpecs/NFTTemplate.js

```diff
@@ -462,6 +456,17 @@ const NFTTemplateSpec = {
               "type": "checkbox",
               "no_localize": true
             },
+            {
+              "name": "pack_generator",
+              "type": "select",
+              "default_value": "random",
```

Comment:
Are we planning to change the default once it's implemented?

Reply:
I don't know - I think we need to wait and see what ends up being the prevailing use case.